pkg/apis/sealedsecrets/v1alpha1/sealedsecret_expansion.go

```diff
@@ -277,7 +277,7 @@ func (s *SealedSecret) Unseal(codecs runtimeserializer.CodecFactory, privKeys ma
 		for key, value := range s.Spec.EncryptedData {
 			valueBytes, err := base64.StdEncoding.DecodeString(value)
 			if err != nil {
-				return nil, err
+				errs = append(errs, multierror.Tag(key, err))
```

Comment:
If there's an error here we should continue to the next item.

```suggestion
				errs = append(errs, multierror.Tag(key, err))
				continue
```

Reply:
Good catch. I added another assertion to ensure that invalid base64 does not trigger a decrypt error.